ecs-cli/modules/cli/configure/configure_test.go

```diff
@@ -124,7 +134,7 @@ func TestDefaultProfile(t *testing.T) {
 }
 
 func TestConfigureProfile(t *testing.T) {
-	config1 := createProfileConfig(profileName, awsAccessKey, awsSecretKey)
```

Comment:
is "createProfileConfig()" old function in use anywhere else? if not, could it be safely removed?

Reply:
It does appear to be used elsewhere in the file, unless I'm missing something...

Reply:
D'oh! Guess I left some of the existing code unexpanded when I looked. I see now it's used on (new) lines 105 & 106.